docs/cody/clients/feature-reference.mdx

```diff
@@ -17,6 +17,8 @@ Here's a feature parity matrix that compares the capabilities of Cody Clients ac
 | Clear chat history          | ✓           | ✓             | x          | ✓       |
 | LLM Selection               | ✓           | ✓             | x          | x       |
 | Enhanced  Context Selection | ✓           | ✓             | x          | x       |
+| `@-file` support for context | ✓           | ✓             | x          | x       |
+| `@#-symbol` support for context | ✓           | x             | x          | x       |
```

Comment:
Can we also add here that only VSC supports Ollama right now (in experimental)? I don't think JB gets this out of the box, does it @philipp-spiess ?

Reply:
Yes that's correct although we could expose the configuration for it if there's interest. I think that's mostly on the JB team if they would think exposing this setting makes sense.

Reply:
Done 👍